sprint_1/src/laura/uebungsprojekt/aufgaben/objektorientierung/Person.java

```diff
@@ -1,4 +1,4 @@
-package laura.uebungsprojekt.aufgaben;
+package laura.uebungsprojekt.aufgaben.objektorientierung;
```

Comment:
Ich hätte ja eher die "alten" Aufgaben in einen eigenen Folder geschoben, aber so ist denke ich auch OK, wenn du damit klarkommst